iib/workers/tasks/opm_operations.py

```diff
@@ -1034,7 +1040,16 @@ def opm_create_empty_fbc(
 
     # Remove all the operators from the index
     set_request_state(request_id, 'in_progress', 'Removing operators from index image')
-    _opm_registry_rm(index_db_path=index_db_path, operators=operators, base_dir=temp_dir)
+    try:
+        _opm_registry_rm(index_db_path=index_db_path, operators=operators, base_dir=temp_dir)
+    except IIBError as e:
+        if 'Error deleting packages from database' in str(e):
+            log.info('Enable permissive mode for opm registry rm', from_index)
+            _opm_registry_rm(
+                index_db_path=index_db_path, operators=operators, base_dir=temp_dir, permissive=True
+            )
+        else:
+            raise e
```

Comment:
This should probably be raise IIBError?

Reply:
It is IIBError. It just propagates the exception which was received.

Reply:
ah, the callback..makes sense!